crates/ark-contracts/arkchain/tests/order/test_order_v1.cairo

```diff
@@ -11,78 +11,98 @@ use arkchain::order::types::RouteType;
 #[test]
 fn test_validate_common_data_with_valid_order() {
     let (order_listing, _, _, _) = setup();
-    let block_timestmap: u64 = 1699556828;
-    let result = order_listing.validate_common_data(block_timestmap);
+    let block_timestamp: u64 = 1699556828; // 09/11/2023 20:07:08
+    let result = order_listing.validate_common_data(block_timestamp);
     assert(result.is_ok(), 'Invalid result');
 }
 
 #[test]
 fn should_returns_invalid_order_with_zero_quantity() {
     let (order_listing, _, _, _) = setup();
-    let block_timestmap: u64 = 1699556828;
+    let block_timestamp: u64 = 1699556828;
 
     let mut invalid_order = order_listing.clone();
     invalid_order.quantity = 0;
-    let result = invalid_order.validate_common_data(block_timestmap);
+    let result = invalid_order.validate_common_data(block_timestamp);
     assert(!result.is_ok(), 'zero quantity');
 }
 
 #[test]
 fn should_returns_invalid_order_with_zero_salt() {
     let (order_listing, _, _, _) = setup();
-    let block_timestmap: u64 = 1699556828;
+    let block_timestamp: u64 = 1699556828;
 
     let mut invalid_order = order_listing.clone();
     invalid_order.salt = 0;
-    let result = invalid_order.validate_common_data(block_timestmap);
+    let result = invalid_order.validate_common_data(block_timestamp);
     assert(!result.is_ok(), 'zero salt');
 }
 
 #[test]
 fn should_returns_invalid_order_with_zero_token_id() {
     let (order_listing, _, _, _) = setup();
-    let block_timestmap: u64 = 1699556828;
+    let block_timestamp: u64 = 1699556828;
 
     let mut invalid_order = order_listing.clone();
     invalid_order.token_id = 0;
```

Comment:
`token_id` can be equal to `0`. Also... I am pretty sure that token_id 0 is a valid token_id, no?